src/stacks-hierarchy/Root_ParkingViolationsReporting/Root_ScooterContactOperatorScreen.tsx

```diff
@@ -0,0 +1,285 @@
+import React, {useState} from 'react';
+import {KeyboardAvoidingView, ScrollView, View} from 'react-native';
+import {StyleSheet, useThemeContext} from '@atb/theme';
+import {ThemeText} from '@atb/components/text';
+import {useTranslation} from '@atb/translations';
+import {RootStackScreenProps} from '@atb/stacks-hierarchy';
+import {
+  PhoneInputSectionItem,
+  RadioGroupSection,
+  Section,
+  TextInputSectionItem,
+} from '@atb/components/sections';
+import {ContentHeading} from '@atb/components/heading';
+import {useVehicle} from '@atb/mobility/use-vehicle';
+import {ContactOperatorTexts} from '@atb/translations/screens/ContactOperator';
+import {
+  SendSupportRequestBody,
+  SendSupportRequestBodySchema,
+  SupportType,
+} from '@atb/api/types/mobility';
+import {useAuthContext} from '@atb/auth';
+import {getParsedPrefixAndPhoneNumber} from '@atb/utils/phone-number-utils';
+import {Button} from '@atb/components/button';
+import {isValidPhoneNumber} from 'libphonenumber-js';
+import {MessageInfoBox} from '@atb/components/message-info-box';
+import {useSendSupportRequestMutation} from '@atb/mobility/queries/use-send-support-request-mutation';
+import {useActiveShmoBookingQuery} from '@atb/mobility/queries/use-active-shmo-booking-query';
+import {getCurrentCoordinatesGlobal} from '@atb/GeolocationContext';
+import {useProfileQuery} from '@atb/queries';
+import {isValidEmail} from '@atb/utils/validation';
+import {FullScreenHeader} from '@atb/components/screen-header';
+import {getThemeColor} from './components/ScreenContainer';
+
+const supportItems: SupportType[] = [
```

Comment:
This might be changed to:

const supportItems = Object.values(SupportType) as SupportType[];